providers/standard/tests/unit/standard/sensors/test_time_delta.py

```diff
@@ -172,8 +172,9 @@ def setup_method(self):
     def test_timedelta_sensor(self, mocker, should_defer):
         defer_mock = mocker.patch(DEFER_PATH)
         delta = timedelta(hours=1)
-        with pytest.warns(AirflowProviderDeprecationWarning):
-            op = TimeDeltaSensorAsync(task_id="timedelta_sensor_check", delta=delta, dag=self.dag)
```

Comment:
we might still want the deprecation warning? And not to fully remove the sensor - the deprecation said next **_major version_**